client/src/pages/Home.js

```diff
@@ -1,42 +1,13 @@
-import { useEffect, useState } from "react";
-import { Link } from "react-router-dom";
-
 import "./Home.css";
-import logo from "./logo.svg";
+import User from "../component/User";
+import MilestoneTable from  "../component/MilestoneTable";
 
 export function Home() {
-	const [message, setMessage] = useState("Loading...");
-
-	useEffect(() => {
-		fetch("/api")
-			.then((res) => {
-				if (!res.ok) {
-					throw new Error(res.statusText);
-				}
-				return res.json();
-			})
-			.then((body) => {
-				setMessage(body.message);
-			})
-			.catch((err) => {
-				console.error(err);
-			});
-	}, []);
 
 	return (
-		<main role="main">
-			<div>
-				<img
-					className="logo"
-					data-qa="logo"
-					src={logo}
-					alt="Just the React logo"
-				/>
-				<h1 className="message" data-qa="message">
-					{message}
-				</h1>
-				<Link to="/about/this/site">About</Link>
-			</div>
+		<main className="main_container" role="main">
```

Comment:
the main tag shouldnt be given a role as it already has the role main

Reply:
it was just there by default and I wasn't sure if its broken something if i delete it, I will remove it on next pull request